app/cli-client/src/app/proto/hello_world_service_pb.d.ts

```diff
@@ -0,0 +1,45 @@
+// package: com.google.startuposexample.app.localserver.service
```

Comment:
Why do you need to add these files to the repo and not generate them in gulp?
What does `npm run protoc` do then?

Reply:
My bad, I forgot to add the files to `.gitignore`